tools/AzureRM.Example.psm1

```diff
@@ -147,3 +147,5 @@ if ($Env:ACC_CLOUD -eq $null)
         }
     }
 }
+
+%COMMAND-NOT-FOUND%
```

Comment:
Is this expected?

Reply:
This works as a placeholder. When building a module, `UpdateModules.psm1` will replace this with real code (if module is Az.Accounts) or simply remove it.